### ISSUE TITLE
Altmetrics shortcode

### ISSUE THREAD

**Issue by patricktokeeffe** [washingtonstateuniversity/WSUWP-Plugin-Embeds#82]
Almetrics is a service for showcasing scholarly content but WSU WordPress policy prevents users from including the third-party script necessary to use the service. Please add a shortcode for Almetrics Badge embeds: https://www.altmetric.com/products/altmetric-badges/

> <http://api.altmetric.com/embeds.html>:
>
> 1. Add the following line of code anywhere on an HTML page: `<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>`
> 2. Add a div element specifying a DOI (digital object identifier), arXiv ID, Handle, PubMed ID, ISBN, URI or Altmetric ID wherever you want a badge to appear: `<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/nature.2012.9872"></div>`
> 
> Replace the contents of data-doi with the DOI of the article you want the badge to represent: alternatively you can use a data-arxiv-id attribute containing an arXiv ID, data-handle attribute containing a Handle, data-isbn attribute containing an ISBN, data-uri attribute containing a URI or data-pmid attribute containing a PubMed ID.

Minimum tag support for a functional shortcode:
* `data-doi` (technically optional but should probably be mandatory unless these alternatives are also supported and at least one is present: `data-arxiv-id`, `data-handle`, `data-isbn`, `data-uri`)
* `data-link-target` (to specify open links in same/new window)
* `data-badge-type` (to specify badge shape/size)

Other tags users will prob want to use:
* *see alternate data-\* handles above*
* `data-badge-popover`
* `data-badge-details`
* `data-no-score`
* `data-hide-less-than`
* `data-condensed`

Alternate tag formats are [also available](http://api.altmetric.com/embeds.html#technicalinfo).

**Comment by nicdford** [comment]
@patricktokeeffe would you be interested in making a PR for this? If not, no worries, @danbleile can let you know when we might be able to fit this in our development schedule.

**Comment by patricktokeeffe** [comment]
@nicdford Yes, I'd be happy to submit a draft but I don't have an environment for testing. The PowerBI looks like a good shortcode to model on.

**Comment by nicdford** [comment]
> @nicdford Yes, I'd be happy to submit a draft but I don't have an environment for testing. The PowerBI looks like a good shortcode to model on.

Sounds good! Any help on the coding front will definitely speed things along :)
